lib/Parse/ParseDecl.cpp

```diff
@@ -2503,7 +2503,7 @@ static std::optional<Identifier> parseSingleAttrOptionImpl(
   };
   bool isDeclModifier = DeclAttribute::isDeclModifier(DK);
 
-  if (!P.Tok.is(tok::l_paren)) {
+  if (!P.Tok.isFollowingLParen()) {
```

Comment:
Unrelated to this change but I've always disliked the name `isFollowingLParen`, to me it reads as "is token following an l-paren", when really it's "is token a following-l-paren". Maybe we should just call it `isSameLineLParen`?